parity/db/rocksdb/blooms.rs

```diff
@@ -63,7 +63,7 @@ pub fn migrate_blooms<P: AsRef<Path>>(path: P, config: &DatabaseConfig) -> Resul
 	// 1u8 -> TraceDBIndex::BloomGroups
 	// 0u8 -> level 0
 	let trace_blooms_iterator = db.key_value()
-		.iter_from_prefix(4, &[1u8, 0u8])
+		.iter_with_prefix(4, &[1u8, 0u8])
 		.filter(|(key, _)| key.len() == 6)
 		.take_while(|(key, _)| {
 			key[0] == 1u8 && key[1] == 0u8
```

Comment:
the `take_while` can be removed here and above

Reply:
Now that I think about it: why do we even need to keep this migration? There aren't any supported clients out there that don't have their trace blooms migrated already, are there?

Reply:
Right, the migration was introduced 2.0, so we can assume that nobody runs 1.x anymore
https://ethernodes.org/client/parity-ethereum

Reply:
Cool. Deleting.